### PR TITLE
Alt Text Inclusions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing alt text.

### CODE REVIEW

File: iis/publish/deploying-application-packages/umbraco.md [hunk 1]
```diff
@@ -76,40 +76,40 @@ To use WebMatrix to install this application, complete these steps:
...
 9. Click **Continue**. 
 
     Once publishing is complete, the Deployment Details dialog box appears.  
-   [![](umbraco/_static/image30.jpg)](umbraco/_static/image29.jpg)
+   ![Screenshot of the Deployment Details dialog box.](umbraco/_static/image29.jpg)
 10. Click **OK**.  
   
 The Umbraco Configuration Wizard appears. From this window, you can configure the Umbraco application and view additional information about the Umbraco application.  
-[![](umbraco/_static/image32.jpg)](umbraco/_static/image31.jpg)
+![Screenshot of the Umbraco Configuration Wizard, reading Thank you for choosing umbraco.](umbraco/_static/image31.jpg)
```
Comment (MeeraDi) [inline iis/publish/deploying-application-packages/umbraco.md:112]: Should "umbraco" be capitalized?

File: iis/web-hosting/hosting-the-microsoft-web-platform-on-amazon-ec2/amazon-ec2-microsoft-web-platform-images-wordpress.md [hunk 2]
```diff
@@ -41,23 +41,23 @@ Just be patient and wait for a little longer.
 
 Once you are signed up successfully you will have access to the Amazon EC2 Management Console. The URL is [https://console.aws.amazon.com/ec2/home](https://console.aws.amazon.com/ec2/home).
 
-[![](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image2.png)](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image1.png)
+![Screenshot of the Amazon EC2 tab with the U S West option being highlighted.](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image1.png)
```
Comment (MeeraDi) [inline iis/web-hosting/hosting-the-microsoft-web-platform-on-amazon-ec2/amazon-ec2-microsoft-web-platform-images-wordpress.md:44]: E C 2

File: iis/web-hosting/hosting-the-microsoft-web-platform-on-amazon-ec2/amazon-ec2-microsoft-web-platform-images-wordpress.md [hunk 3]
```diff
@@ -69,17 +69,19 @@ Now we are ready to launch an EC2 DotNetNuke instance.
 ### Launching an DotNetNuke Instance
 
 Click the "Instances" menu item on the left hand side and click the "Launch Instance" button.   
-[![](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image10.png)](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image9.png)
+![Screenshot of the Launch Instance option.](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image9.png)
 
 Now select "Community Images" in the resulting dialog.
 
-[![](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image12.png)](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image11.png)
+![Screenshot of the Quick Start, My A M Is, and Community A M Is tabs.](amazon-ec2-microsoft-web-platform-images-wordpress/_static/image11.png)
```
Comment (MeeraDi) [inline iis/web-hosting/hosting-the-microsoft-web-platform-on-amazon-ec2/amazon-ec2-microsoft-web-platform-images-wordpress.md:76]: A M I's (here and the next one)

**Comment by dereklh7** [comment]
@Rick-Anderson @mcleblanc 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**